stages/apps/lead/harbor.tf

```diff
@@ -1,15 +1,24 @@
+locals {
+  ingress_annotations = {
```

Comment:
I wonder if these should be moved somewhere more generic, since they could be used for other deployments as well.  If the intention is to only use these annotations for the harbor ingress, then this should probably be renamed to `harbor_ingress_annotations`, since this local variable can be accessed within other files in this module as well.

Reply:
That is a really good point. Yeah we will likely have ingress annotations for various apps, but they will usually be specific, so ill need to update this.